packages/at_commons/lib/src/verb/enroll_params.g.dart

```diff
@@ -1,5 +1,4 @@
 // GENERATED CODE - DO NOT MODIFY BY HAND
-// dart run build_runner build to generate this file
```

Comment:
please add back the builder line

Reply:
change made as part of [feddc9e](https://github.com/atsign-foundation/at_libraries/pull/530/commits/feddc9e80d5327223fa0a5fed5f6ff9a3173fe56)